src/components/TimePicker/TimePicker.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import styled from 'styled-components';
+import PropTypes from 'prop-types';
+import Input from 'components/Input';
+import RadioGroup from 'components/RadioGroup';
+const RadioButton = RadioGroup.Button;
+import moment from 'moment';
+import Container from './styles/Container';
+import RadioContainer from './styles/RadioContainer';
+import StyledInput from './styles/StyledInput';
+
+class TimePicker extends React.PureComponent {
+  static propTypes = {
+    /**
+     * TODO: Rewrite this description
+     */
+    value: PropTypes.oneOfType([PropTypes.object, PropTypes.number]),
+    /**
+     * Time format to pass to moment to format the string. Probably hh:mm or hh:mm:ss.
+     */
+    timeFormat: PropTypes.string,
+  };
+
+  static defaultProps = {
+    value: undefined,
+    timeFormat: 'hh:mm',
+  };
+
+  state = {
+    internalValue: undefined,
+  };
+
+  handleOnChange = ({ value, amSelected }) => {
+    if (!value) {
+      return;
+    }
+    const timeValue = moment(value, this.props.timeFormat);
+    const hours = amSelected === timeValue.hours() >= 12 ? 12 : 0;
```

Comment:
This statement is pretty confusing to read. `amSelected equals timevalue hours greater than or equal to twelve, then twelve else zero`? Seems like there's some JS order-of-operations needed to parse... would be preferable to simplify and break down.